diff --git a/Makefile.am b/Makefile.am
index 66d757b..cd07035 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -20,9 +20,8 @@
 ## Process this file with automake to produce Makefile.in
 
 EXTRA_DIST = bootstrap BUGS FAQ README.translators \
-             config/COPYING.davfs2 config/INSTALL.davfs2 \
-             config/gnulib-cache.m4
-SUBDIRS = gl glpo po etc man src
+             config/COPYING.davfs2 config/INSTALL.davfs2
+SUBDIRS = po etc src
 ACLOCAL_AMFLAGS = -I config
 doc_DATA = AUTHORS BUGS ChangeLog COPYING FAQ INSTALL NEWS README \
            README.translators THANKS TODO
