diff --git a/Makefile.am b/Makefile.am
index 335b3c5..7924dd2 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -21,7 +21,7 @@
 
 EXTRA_DIST = bootstrap BUGS FAQ README.translators \
              config/COPYING.davfs2 config/INSTALL.davfs2
-SUBDIRS = po etc man src
+SUBDIRS = etc src
 ACLOCAL_AMFLAGS = -I config
 doc_DATA = AUTHORS BUGS ChangeLog COPYING FAQ INSTALL NEWS README \
            README.translators THANKS TODO
